cisagov/windows-server-packer
# first-commits: initialize ami with base Windows Server 2022

Initialize project with a base windows image and a bootstrap winrm file for packer to be able to connect to the instance via WinRM.

## 🗣 Description ##
This PR only contains the base Windows Server 2022 image for initializing the project.
The Windows Administrator password for this AMI is set by AWS SSM Parameter Store.

## 💭 Motivation and context ##
This project can eventually be used to be able to create machine images of Windows Commando VM.

## 🧪 Testing ##
Testing locally and within this pull request to get all hooks to pass and successfully create an AMI.

Update:
Successfully tested the AMI by doing these steps:
- Launch an EC2 instance with the AMI (using the same subnet that was used to create the AMI)
- Update Security Group and Network ACL Inbound Rules to allow only my ip address
- Create an Internet Gateway and add it to the Subnet's Routing Table to be able to access it publicly.
- Successfully logged in using the password specified by the WinRM bootstrap launch script using Windows Remote Desktop
- Verified that Windows Defender has been disabled and removed as expected per the provisioner step in the Packer script.

## ✅ Pre-approval checklist ##


## ✅ Pre-merge checklist ##





## ✅ Post-merge checklist ##
